src/XComponent.Functions/Utilities/SerializationHelper.cs

```diff
@@ -39,9 +40,8 @@ public static string SerializeObject(object obj)
             }
             catch (Exception e)
             {
-                Debug.WriteLine(e);
+                throw new SerializationException($"Couldn't serialize object {obj} : {e}");

```

Comment:
do not put the exception in the string, pass it to the constructor

https://msdn.microsoft.com/fr-fr/library/d35hfdat(v=vs.110).aspx